src/components/ListDetail/ListDetail.tsx

```diff
@@ -301,6 +323,25 @@ export const ListDetail = ({
       <div className={styles['list-detail__body']}>
         {childrenWithProps[activeIndexState]}
       </div>
+      <Drawer
```

Comment:
one challenge with using a Drawer here: the Drawer component sets a width of 28rem (Drawer.module.css:55) but for this implementation we want the width to be 100%. Can Drawer component be modified to accept width variants, or at least a full-width variant?